zapisy/apps/enrollment/courses/models/tags.py

```diff
@@ -0,0 +1,35 @@
+from django.db import models
+
+
+class BaseTag(models.Model):
+    short_name = models.CharField(max_length=50, verbose_name='nazwa skrócona')
+    full_name = models.CharField(max_length=250, verbose_name='nazwa pełna')
+    description = models.TextField(verbose_name='opis')
+
+    class Meta:
+        abstract = True
+
+    def __str__(self):
+        return f"{self.short_name} ({self.full_name})"
+
+    def serialize_for_json(self):
+        return {
+            'id': self.pk,
+            'short_name': self.short_name,
+            'full_name': self.full_name,
+            'description': self.description,
+        }
+
+
+class ThematicTag(BaseTag):
+    class Meta:
+        verbose_name = 'Tag tematyczny (I st.)'
+        verbose_name_plural = 'Tagi tematyczne (I st.)'
+        app_label = 'courses'
+
+
+class SpecialistTag(BaseTag):
+    class Meta:
+        verbose_name = 'Tag specjalistyczny (II st.)'
+        verbose_name_plural = 'Tagi specjalistyczne (II st.)'
```

Comment:
To może rewolucyjne, ale przy okazji sprawmy, żeby w systemie posługiwać się językiem polskim (i jednocześnie stosować nomenklaturę pojawiającą się w programie studiów) – plus oczywiście analogiczne zmiany w innych plikach.
```suggestion
        verbose_name = "Znacznik tematyczny (I st.)"
        verbose_name_plural = "Znaczniki tematyczne (I st.)"
        app_label = 'courses'


class SpecialistTag(BaseTag):
    class Meta:
        verbose_name = "Znacznik specjalistyczny (II st.)"
        verbose_name_plural = "Znaczniki specjalistyczne (II st.)"
```
(Podwójne cudzysłowy stosujemy dla napisów przeznaczonych "dla ludzkich oczu", a pojedyncze – dla tych przetwarzanych wewnętrznie w kodzie.)